CanalTP/NavitiaSDKUX_ios
# StylizedComponent : Unit Tests

What is it for ?
- Setting up unit tests for StylizedComponent in order to refactorize it afterwards

How to validate ?
- Go to Example
- `pod install`
- Launch unit tests in Example/Tests/Components/Primitive
- Check all tests are green
